scripts/test/unit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+go test -tags daemon -v $@
```

Comment:
A mess of bash scripts. It contributes to a horrid build experience and is very hard to modify.

This particular example is absurd. Why is this in its own file?